src/functions/findFirst.js

```diff
@@ -78,6 +93,32 @@ export const findFirst = wrap({
         }
         return undefined;
     },
+    tests: process.env.NODE_ENV !== "development" ? undefined : {
+        "basicUsage": (hi) => {
+            const seq = hi("a string to be searched").findFirst("to be");
+            hi.assert(seq, (r) => r.index === 9);
```

Comment:
This assert isn't correct

The first attribute is a condition evaluated for truthiness and the second is an error message for when it fails